src/Forms/Shared/Samples/Data/EditFeatureLinkedAnnotation/EditFeatureLinkedAnnotation.xaml.cs

```diff
@@ -23,7 +23,7 @@ namespace ArcGISRuntimeXamarin.Samples.EditFeatureLinkedAnnotation
         name: "Edit features with feature-linked annotation",
         category: "Data",
         description: "Edit feature attributes which are linked to annotation through an expression.",
-        instructions: "Pan and zoom the map to see that the text on the map is annotation, not labels. Tap one of the address points to update the house number .GetGeodatabaseAnnotationTable((AD_ADDRESS) and street name (ST_STR_NAM). Tap one of the dashed parcel polylines and tap another location to change its geometry. NOTE: Selection is only enabled for points and straight (single segment) polylines.",
+        instructions: "Pan and zoom the map to see that the text on the map is annotation, not labels. Tap one of the address points to update the house number (AD_ADDRESS) and street name (ST_STR_NAM). Tap one of the dashed parcel polylines and tap another location to change its geometry. NOTE: Selection is only enabled for points and straight (single segment) polylines.",
```

Comment:
Unrelated fix, should be in a seperate PR or commit.

Reply:
Good catch here.